docs/node/admin/kwil-admin/setup/testnet.mdx

```diff
@@ -46,23 +46,26 @@ kwil-admin setup testnet -v 4 -o ./output --hostnames 192.168.10.2 192.168.10.3
 ### Options
 
 ```
-      --alloc allocFlag           account=amount pairs of genesis account allocations (default map[])
-  -i, --block-interval duration   shortest block interval in seconds (default 6s)
-      --chain-id string           chain ID to use for the genesis file
-      --config string             path to a config file to use as a template for all nodes
-      --gas                       enable gas
-  -h, --help                      help for testnet
-      --hostname-prefix string    prefix for hostnames of nodes
-      --hostname-suffix string    suffix for hostnames of nodes
-      --hostnames strings         override all hostnames of the nodes (list of hostnames must be the same length as the number of nodes)
-      --join-expiry int           number of blocks before a join request expires (default 14400)
-      --node-dir-prefix string    prefix for the node directories (node results in node0, node1, ...) (default "node")
-  -n, --non-validators int        number of non-validators to generate
-  -o, --output-dir string         parent directory for all of generated node folders (default "./.testnet")
-  -p, --p2p-port int              p2p port for nodes (default 26656)
-      --starting-ip string        starting IP address for nodes (default "172.10.100.2")
-  -v, --validators int            number of validators to generate (default 3)
-      --without-nonces            disable account nonces
+      --alloc allocFlag                account=amount pairs of genesis account allocations (default map[])
+  -i, --block-interval duration        shortest block interval in seconds (default 6s)
+      --chain-id string                chain ID to use for the genesis file
+      --config string                  path to a config file to use as a template for all nodes
+      --gas                            enable gas
+  -h, --help                           help for testnet
+      --hostname-prefix string         prefix for hostnames of nodes
+      --hostname-suffix string         suffix for hostnames of nodes
+      --hostnames strings              override all hostnames of the nodes (list of hostnames must be the same length as the number of nodes)
+      --join-expiry int                number of blocks before a join request expires (default 14400)
+      --node-dir-prefix string         prefix for the node directories (node results in node0, node1, ...) (default "node")
+  -n, --non-validators int             number of non-validators to generate
+  -o, --output-dir string              parent directory for all of generated node folders (default "./.testnet")
+  -p, --p2p-port int                   p2p port for nodes (default 26656)
+      --snapshots.enable               enables db snapshots
+      --snapshots.heights uint         Heights to snapshot the db (default 10000)
+      --snapshots.max-snapshots uint   Maximum number of snapshots to store in the device (default 3)
```

Comment:
These descriptions are capitalized (Heights / Maxiumum) unlike the others and that stands out.  Best to stay consistent.